.github/workflows/auto-cleanup-bot.yml

```
@@ -32,7 +32,7 @@ jobs:
 
       - name: Lint markdown files
         run: |
-          yarn content fix-flaws
+          # yarn content fix-flaws
```

We discussed this internally, and we want to implement this in rari, but for now we recommend to fall-back to `content:legacy`:

```suggestion
          yarn content:legacy fix-flaws
```

---

Yes the command is still useful: https://github.com/mdn/content/pull/38498